ansible/configs/kni-osp/pre_software.yml

```diff
@@ -0,0 +1,51 @@
+---
+
+- name: Step 003 - Configure all hosts with Repositories, Common Files and Set environment key
+  hosts:
+  - utility
+  become: true
+  gather_facts: False
+  tags:
+  - step004
+  - common_tasks
+  roles:
+  - { role: "set-repositories",       when: 'repo_method is defined' }
+  - { role: "common",                 when: 'install_common | bool' }
+  - { role: "set_env_authorized_key", when: 'set_env_authorized_key | bool' }
+  tasks:
+    - name: Add GUID to /etc/skel/.bashrc
+      lineinfile:
+        path: "/etc/skel/.bashrc"
+        regexp: "^export GUID"
+        line: "export GUID={{ guid }}"
+
+    - name: Create hosts file
+      template:
+        src: "./files/hosts.j2"
+        dest: "/etc/hosts"
+        owner: "root"
+        force: yes
+        mode: 0644
+
+- name: Configuring Bastion Hosts
+  hosts: utility
```

Comment:
Why is a play to target bastions targeting utility instead now?

Reply:
merging bastion and utility roles got me to keep only the utility role as bastion host was never really used (hence the merging)